runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1306,6 +1371,59 @@ func s:GotoAsmwinOrCreateIt()
   endif
 endfunc
 
+func s:GetVariablesWindow()
+  if exists('g:termdebug_config')
+    return get(g:termdebug_config, 'variables_window', 0)
+  endif
+  if exists('g:termdebug_disasm_window')
+    return g:termdebug_variables_window
+  endif
+  return 0
+endfunc
+
+func s:GetVariablesWindowHeight()
+  if exists('g:termdebug_config')
+    return get(g:termdebug_config, 'variables_window_height', 0)
+  endif
+  if exists('g:termdebug_variables_window') && g:termdebug_variables_window > 1
+    return g:termdebug_variables_window
+  endif
+  return 0
+endfunc
+
+func s:GotoVariableswinOrCreateIt()
+  if !win_gotoid(s:varwin)
+    if win_gotoid(s:sourcewin)
+      exe 'rightbelow new'
+    else
+      exe 'new'
+    endif
+
+    let s:varwin = win_getid(winnr())
+
+    setlocal nowrap
+    setlocal number
+    setlocal noswapfile
+    setlocal buftype=nofile
+    setlocal modifiable
+
+    let varbuf = bufnr('Termdebug-variables-listing')
+    if varbuf > 0
+      exe 'buffer' . varbuf
+    else
+      exe 'file Termdebug-variables-listing'
```

Comment:
looks here, perhaps better prefix with `silent`, this should be not exist when auto init to open it.
silent file Termdebug-variables-listing

Reply:
Is This something that needs to be changed for all windows then? Because I just followed the existing practice for the existing assembly window.

https://github.com/vim/vim/blob/6c313bbb043745a60a7ee30a78edb907260b764a/runtime/pack/dist/opt/termdebug/plugin/termdebug.vim#L1285-L1289